cscs-checks/microbenchmarks/stream/stream.py

```diff
@@ -130,5 +126,3 @@ def setup(self, partition, environ, **job_opts):
             self.reference = self.stream_bw_reference[envname]
         except KeyError:
             self.reference = {'*': {'triad': (0.0, None, None, 'MB/s')}}
```

Comment:
```suggestion
            pass
```

You don't need to set the default reference anymore.